### DISCUSSION TITLE
Custom signin error page that falls back to core

### DISCUSSION THREAD

**Discussion by sdoxsee** [nextauthjs/next-auth#6590] [Help]
### Question 💬

I want to have a custom signin error page for a specific error code but fall back to the default for anything else. I want to delegate to the default I have a chance to handle it.

However, when I run/build I get the following error: 
```
Module not found: Package path ./core/pages/signin is not exported from package <project_dir>/node_modules/next-auth (see exports field in <project_dir>/node_modules/next-auth/package.json)
```

Custom Signin Page
```
import type { SignInServerPageParams } from 'next-auth/core/pages/signin';
import SigninPage from 'next-auth/core/pages/signin';

const CustomSigninPage = (params: SignInServerPageParams) => {
  if (params.error === 'OAuthCallback') {
    return <>My own content</>
  }
  return <SigninPage {...params} />
}

export default CustomSigninPage
```

### How to reproduce ☕️

https://codesandbox.io/p/sandbox/zen-varahamihira-5pj7tp

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment]
You could do this with Advanced Initialization and a redirect https://next-auth.js.org/configuration/initialization#advanced-initialization
